### ISSUE TITLE
Possible new feature: the autorotation tool

### ISSUE THREAD

**Issue by Zonkil9** [dewberryants/asciiMol#6]
Hello!

This is a very cool project! Although I'm missing one feature that would be useful: the autorotation tool that would allow for the continuous rotation of a molecule. Is it possible to add it in the future?

**Comment by dewberryants** [comment]
Hey! Yes, in fact I've been working on a few things that will make the rotation and navigation a little easier (such as centering on center of coordinates and rotating to principal axes). So this should fit right in.

**Comment by dewberryants** [comment]
I have implemented an auto-rotation in #7 , if you want to have a look at it. It is hardcoded to rotate 1.6 degrees per two frames, which at 60 fps is 48 degrees a second. Not sure if this is universally applicable, it just look smooth and nice on my terminal.

Be aware that on terminals that don't draw quickly enough, this will flicker horrendously (tested it on my TTY)

EDIT: The feature can be accessed via the F[1-3] keys
EDIT2: I found that rotating at 60 fps and 30 fps respectively looked almost identical. So I think 30 fps should be sufficient.
